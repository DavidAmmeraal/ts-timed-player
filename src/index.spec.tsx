/**
 * @jest-environment jsdom
 */

 describe('index.tsx', () => {
  it('should render the App connected to redux in the #root element of the HTML page', () => {
    const rootElement = document.createElement('div');
    rootElement.id = 'root';
    document.body.appendChild(rootElement);
    require('./index');
    expect(rootElement).toMatchSnapshot();
  })
 });