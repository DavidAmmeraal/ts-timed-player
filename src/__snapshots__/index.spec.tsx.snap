// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.tsx should render the App connected to redux in the #root element of the HTML page 1`] = `
<div
  id="root"
>
  <span>
    LOADING
  </span>
</div>
`;
